near/src/runtime.rs

```diff
@@ -1053,8 +1065,14 @@ impl RuntimeAdapter for NightshadeRuntime {
         let trie_changes = Trie::combine_state_parts(&state_root, &parts)
             .expect("combine_state_parts is guaranteed to succeed when each part is valid");
         // TODO clean old states
```

Comment:
does `clean old states` now relates to erasing data from ColFlatState, not trie?